Userland/Libraries/LibGUI/InputBox.h

```diff
@@ -16,14 +16,14 @@ class InputBox : public Dialog {
 public:
     virtual ~InputBox() override;
 
-    static int show(Window* parent_window, String& text_value, StringView const& prompt, StringView const& title, StringView const& placeholder = {});
+    static int show(Window* parent_window, String& text_value, StringView const& prompt, StringView const& title, StringView const& placeholder = {}, bool is_password = false);
```

Comment:
Let's at least make the user-facing API use an enum instead of a bool here (e.g. something like `InputType::{Password, Plain}`)